Formula/bear.rb

```diff
@@ -24,6 +24,12 @@ class Bear < Formula
   depends_on "spdlog"
   depends_on "sqlite"
 
+  depends_on "gcc" unless OS.mac?
```

Comment:
on_macos do / end

Reply:
```
Formula/bear.rb:27:27: C: Don't use OS.mac?; homebrew/core only supports macOS
```
This style message could be improved now to suggest `on_macos` and `on_linux`. If you want to get real crazy, `brew style --fix` could probably automate rewriting it.